packages/jira-go/build.sh

```diff
@@ -0,0 +1,18 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/go-jira/jira
+TERMUX_PKG_DESCRIPTION="Simple jira command line client in Go"
+TERMUX_PKG_LICENSE="Apache-2.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION="1.0.27"
+TERMUX_PKG_SRCURL=https://github.com/go-jira/jira/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=c5bcf7b61300b67a8f4e42ab60e462204130c352050e8551b1c23ab2ecafefc7
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_AUTO_UPDATE=true
+
+termux_step_make() {
```

Comment:
Remove this and instead call `termux_setup_golang` in `termux_step_pre_configure`.

Makefile will be recognised by build system.